token-stakedrop/README.adoc

```diff
@@ -1,70 +1,87 @@
-# KEEP Token Tracker
+# Overview
 
-This is a tool for tracking KEEP Token ownership. It will walk over predefined
-sources of truth and inspect KEEP Token ownership at a specific block number.
+This is a tool for finding KEEP token owners on Ethereum (or on any other 
+EVM-compatible chain). It will walk over predefined sources of truth
+(information) and lookup KEEP token owners at a specific block number.
+The tool generates a Merkle tree used as an input to
+https://github.com/keep-network/keep-core/blob/master/solidity/contracts/token-distribution/TokenDistributor.sol[`TokenDistributor`]
+contract airdropping another ERC20 token with amounts based on KEEP token 
+balances. 
 
-The tool returns a map of addresses with their total KEEP Token holdings.
-
-## Sources Of Truth
+# Sources of truth
 
 Holders of the tokens can be externally owned accounts or contracts. It is
```

Comment:
Holders of KEEP tokens...

Reply:
https://github.com/keep-network/keep-core/pull/2400/commits/4452560d071fd3aa60b3966c7ea79afe12ada6d4